kinto/tests/support.py

```diff
@@ -44,11 +44,6 @@ def _get_test_app(self, settings=None):
     def get_app_settings(self, additional_settings=None):
         settings = cliquet_support.DEFAULT_SETTINGS.copy()
         settings.update(**DEFAULT_SETTINGS)
-        settings['cliquet.cache_backend'] = 'cliquet.cache.memory'
```

Comment:
We should probably keep that so that we can change the default settings without having to manage that.